#build/Publish-All.ps1

```diff
@@ -79,6 +79,15 @@ function Get-PredefinedBranches() {
     })
 }
 
+function Set-Content-UTF8($file, $content) {

```

Comment:
I would prefer `Set-Utf8Content` or `Set-ContentUtf8` - I think PowerShell naming guidelines just use one dash.

Reply:
Thank you! Pushed a fix.